README.md

```diff
@@ -95,6 +107,22 @@ tomcat8_java_opts:
   - -Dfcrepo.binary.directory={{ fcrepo_data_dir }}
 ```
 
+Similarly, the MySQL and Postgresql object persistence configurations also use Tomcat Java Opts. Again, using ansible-tole-tomcat8:
+```
+fcrepo_persistence: jdbc-mysql
+
+fcrepo_db_name: fcrepo
+fcrepo_db_user: fcrepo
+fcrepo_db_host: "127.0.0.1"
+fcrepo_db_port: "3306"
+
+tomcat8_java_opts:
+- -Dfcrepo.mysql.username={{ fcrepo_db_name }}
+- -Dfcrepo.mysql.password={{ fcrepo_db_password }}
+- -Dfcrepo.mysql.host={{ fcrepo_db_host }}
+- -Dfcrepo.mysql.port={{ fcrepo_db_port }}
```

Comment:
Once we have templated the `repository.json`, this information should be removed and the associated lines in the claw-playbook PR should be removed and any needed variables overridden there.